src/pages/Dashboard.tsx

```diff
@@ -1,3 +1,9 @@
 export const Dashboard = () => {
-	return <div>Dashboard</div>;
+	return (
+		<main>
+			<div className="">Dashboard</div>
+			<button><a href="/login">Login</a></button>
```

Comment:
Use Link instead a tag react (Client side) are not good with it please check on https://reactrouter.com/en/main/components/link